anomalib/models/cflow/config.yaml

```
@@ -6,14 +6,14 @@ dataset:
   task: segmentation
   image_size: 256
   train_batch_size: 16
-  test_batch_size: 16
+  eval_batch_size: 16
   inference_batch_size: 16
   fiber_batch_size: 64
   num_workers: 8
   transform_config:
     train: null
-    val: null
-  create_validation_set: false
+    eval: null
+  validation_split_mode: same_as_test # options: [same_as_test, from_test]
```

Do you think if this will be informative enough for the user? Would be need some explanation somewhere?

---

I'll add some description to the docs

---

```suggestion
  val_split_mode: same_as_test # see options at anomalib.data.utils.ValSplitMode
```

[minor]
like this new users would check all the updated options 
e.g. with your other pr there will be a 3rd option